jobs/common/pre-reqs.yaml

```diff
@@ -81,12 +81,23 @@
         sudo snap install microk8s --classic --channel={channel}
         sudo snap install kubectl --classic --channel={channel}
         echo "waiting for microk8s storage to become available"
+
+        NEXT_WAIT_TIME=0
+        until [ $NEXT_WAIT_TIME -eq 30 ] || sudo microk8s enable dns && microk8s status --yaml | grep -q 'dns: enabled'; do
+            sleep $(( NEXT_WAIT_TIME++ ))
```

Comment:
Might be useful here to add a little message saying something like "trying again" (maybe after the 5th try?), since it looks like we're trying 30 times waiting longer each time, so it'll take quite a long time if microk8s quietly fails to enable.

Reply:
It isn't _too_ long of a wait, 1/2 * 30 * 31 is just under 8 minutes. Remember we have `set -ex` enabled, so it is quite clear from the output what is going on